### PR TITLE
Draft: NNPackage multi models example

### PR DESCRIPTION
**Pull request by hseok-oh**
MANIFEST and file structure for models package.

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
Model example: `compiler/circle-part-value-test`'s `Part_Add_Sqrt_000` example (change name to lowercase)

File: nnpackage/examples/v1.2.0/part_add_sqrt/metadata/MANIFEST [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "major-version" : "1",
+  "minor-version" : "2",
```
Comment (glistening) [inline nnpackage/examples/v1.2.0/part_add_sqrt/metadata/MANIFEST:3]: Please update v1.3.0 since it supports more fields.

**Comment by glistening** [comment]
@hseok-oh Please find my comments:
https://github.com/Samsung/ONE/issues/9123#issuecomment-1128384213
https://github.com/Samsung/ONE/issues/9126#issuecomment-1128377328

**Comment by hseok-oh** [comment]
Updated 
- version: 1.2.0 -> 1.3.0
- name: connection -> model-connect

**Comment by hseok-oh** [comment]
Updated

- Move connect info into `MANIFEST` file (base on https://github.com/Samsung/ONE/issues/9206#issuecomment-1151907553)
- Add `README.md` file 
- Add config file for each model

**Comment by hseok-oh** [comment]
This issue will be handled on #9391

**Pull request closed without merging**